v/vaiavy-chilla.md

```diff
@@ -0,0 +1,10 @@
+# Vaiavy Chilla
```

Comment:
The artist name, and filenames need to be fixed. It looks like Chila only has one 'L'.
Please fix the reference in `v/index.md` too.
Thanks!